### PR TITLE
Feature/learners can add suggestions to learning or remove suggestions

### PR DESCRIPTION
**Pull request by smoga-git**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
 		"@types/pg": "7.4.3",
 		"@types/request": "2.47.0",
 		"@types/session-file-store": "1.2.0",
+		"body-parser": "^1.18.2",
```
Comment (tav) [inline package.json]: Did you mean to add this to `devDependencies` ?

Comment (tav) [reply package.json]: Also, explicit versions please, i.e. remove the `^`.

File: src/ui/controllers/suggestion.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+export async function addToPlan(req: express.Request, res: express.Response) {
+	let uid = req.params.courseId
+	let verb = 'Liked'
+	const course = await catalog.get(uid)
+	const verbId = xapi.lookup(verb)
+
+	if (verbId && course) {
+		try {
+			let resp = await xapi.record(req, course, verbId)
```
Comment (tav) [inline src/ui/controllers/suggestion.ts]: Do we care about the `resp` in this instance?

File: src/ui/controllers/suggestion.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	res: express.Response
+) {
+	let uid = req.params.courseId
+	let verb = 'Disliked'
+	const course = await catalog.get(uid)
+	const verbId = xapi.lookup(verb)
+
+	if (verbId && course) {
+		try {
+			let resp = await xapi.record(req, course, verbId)
```
Comment (tav) [inline src/ui/controllers/suggestion.ts]: Do we care about the `resp` in this instance?

File: src/lib/xapi.ts [hunk 4]
```diff
@@ -123,14 +127,16 @@ export async function record(
 			},
 		}
 	}
-	await send(payload)
+	let resp = await send(payload)
+	return resp
```
Comment (tav) [inline src/lib/xapi.ts]: Can just `return await send(payload)`.

File: package.json (hunk 1 above)
Comment (smoga-git) [reply package.json]: removed

File: src/lib/xapi.ts (hunk 4 above)
Comment (smoga-git) [reply src/lib/xapi.ts]: deliberately returning the response thought it would be generally useful than just firing it off

Comment (smoga-git) [reply src/lib/xapi.ts]: Oh , right you are :D

**Pull request merged**